modules/terraform/apply.go

```diff
@@ -40,8 +41,28 @@ func Apply(t *testing.T, options *Options) string {
 	return out
 }
 
+// ApplyAllTg runs terragrunt apply with the given options and return stdout/stderr. Note that this method does NOT call destroy and
+// assumes the caller is responsible for cleaning up any resources created by running apply.
+func ApplyAllTg(t *testing.T, options *Options) string {
+	out, err := ApplyAllTgE(t, options)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I know this file doesn't have this pattern yet, but we've started shortening this to a one liner: `require.NoError(t, err)`. The import path is `github.com/stretchr/testify/require`. Can you apply to these functions?